Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt

```diff
@@ -32,12 +35,36 @@ class IncidenceCard(parent: ViewGroup) :
         }
 
         with(item.stats as IncidenceStats) {
+
+            incidenceContainer.contentDescription =
+                buildAccessibilityStringForIncidenceCard(item.stats, sevenDayIncidence)
+
             primaryLabel.text = getPrimaryLabel(context)
             primaryValue.text = getLocalizedSpannableString(
                 context,
                 formatStatisticalValue(context, sevenDayIncidence.value, sevenDayIncidence.decimals)
             )
+
+            primaryValue.contentDescription =
+                context.getString(R.string.statistics_explanation_seven_day_incidence_title) + " " +
```

Comment:
for concatenations better use StringBuilder, String.join or String.format 
Its better readable and cleaner.
(also below)

Reply:
Done✔️